src/status_im/native_module/impl/module.cljs

```diff
@@ -205,9 +205,9 @@
     (call-module #(.clearCookies status))
     (call-module #(.clearStorageAPIs status))))
 
-(defn call-web3 [host payload callback]
+(defn call-web3 [payload callback]
   (when status
-    (call-module #(.sendWeb3Request status host payload callback))))
+    (call-module #(.sendWeb3Request status "" payload callback))))
```

Comment:
RCTStatus.m:

RCT_EXPORT_METHOD(sendWeb3Request:

(NSString *)host                  <-- remove this line
                  password:(NSString *)payload
                  callback:(RCTResponseSenderBlock)callback) {
    dispatch_async( dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
        char * result = CallRPC((char *) [payload UTF8String]);
        dispatch_async( dispatch_get_main_queue(), ^{
            callback(@[[NSString stringWithUTF8String: result]]);
        });
    });
}